setup.cfg

```diff
@@ -11,7 +11,7 @@ markers =
 # E203 false positive, see https://github.com/PyCQA/pycodestyle/issues/373
 # W605 false positive until python3.8: https://github.com/PyCQA/pycodestyle/issues/755
 ignore = F999,E203,E501,E128,E124,E402,W503,W504,W605,E731,C901,B007,B306,B009,B010
-exclude = .venv/.git,*/south_migrations/*,node_modules/*,src/sentry/static/sentry/vendor/*,docs/*,src/south/*,examples/*
+exclude = .venv/.git,*/migrations/*,*/south_migrations/*,node_modules/*,src/sentry/static/sentry/vendor/*,docs/*,src/south/*,examples/*
```

Comment:
Aside: should we not move this field into `MANIFEST.in` file?

Reply:
Not sure, this is used by linters, not sure if they will take the `MANIFEST.in` into account?

Reply:
Fairly confident MANIFEST is only for setuptools/distribution packaging

Reply:
Alright then, rescinding.